localization/english/culture/wc_cultural_innovations_l_english.yml

```diff
@@ -32,9 +32,6 @@
  # Early Medieval Era #
  ###################### 
  
- innovation_time_is_money:0 "Time Is Money"
- innovation_time_is_money_desc:0 "The flow of money is like the flow of time, it must be cherished, because without money our time here is meaningless."
-
```

Comment:
Remember the localization guidelines. Don't sort the localization files.

Reply:
Ik the guidelines, but I did it for all languages so that no issues would arise.

Reply:
> Ik the guidelines, but I did it for all languages so that no issues would arise.

For know, but never do it again because there may be ongoing translation on another PR.